src/app/portlets/dot-edit-page/dot-edit-page-resolver.service.spec.ts

```diff
@@ -14,7 +14,7 @@ const activatedRouteSnapshotMock: any = jasmine.createSpyObj<ActivatedRouteSnaps
 ]);
 activatedRouteSnapshotMock.queryParams = {};
 
-describe('ContentTypeResolver', () => {
+describe('ContentTypeEditResolver', () => {
```

Comment:
This file it's called ```dot-edit-page-resolver.service.spec.ts``` but the describe say: ```ContentTypeEditResolver``` I think we're confusing things here, please double check.